pkg/operator/utils/utils.go

```diff
@@ -206,6 +206,16 @@ func GetOperatorConfiguration(kubeClient client.Client, logger log.FieldLogger)
 	return
 }
 
+func GetOperatorLogLevel(kubeClient client.Client, logger log.FieldLogger) (operatorv1.LogLevel, error) {
+
+	conf, err := getOperatorConfiguration(kubeClient, logger)
+	if err != nil {
+		return "", err
+	}
+	// TODO Configuration conflicts
+	return conf.Spec.LogLevel, nil
+}
```

Comment:
`GetOperatorLogLevel` should return `conf.Spec.OperatorLogLevel` I guess

Reply:
You are correct, that is my mistake. I'll fix that up.